examples/counter/src/main.rs

```diff
@@ -24,13 +26,13 @@ impl Component for Model {
         match msg {
             Msg::Increment => {
                 self.value += 1;
-                ConsoleService::log("plus one");
-                true
+                ConsoleService::log("plus one"); // Will output a string to the browser console
+                true // Return true to cause the displayed change to update
             }
             Msg::Decrement => {
                 self.value -= 1;
-                ConsoleService::log("minus one");
-                true
+                ConsoleService::log("minus one"); // Will output a string to the browser console
+                true // Return true to cause the displayed change to update
```

Comment:
I don't think we need the same comment(s) twice.
```suggestion
                ConsoleService::log("minus one");
                true
```